example/train.py

```diff
@@ -0,0 +1,32 @@
+from keras.datasets import cifar10
```

Comment:
@sotanishy can you create a `code_samples` folder in `example/{framework}` based on the framework for which code is written. And put the samples there